### DISCUSSION TITLE
Embed interactive plotly figures in qmd doc

### DISCUSSION THREAD

**Discussion by AakashKumarNain** [quarto-dev/quarto-cli#2478] [Q&A] [answered]
I have an interactive plot generated using `plolty` and saved it as an html file to the disk. Is there a way to render the interactive plot in the `qmd` document>?

**Comment by rgaiacs** [comment, marked as answer]
@AakashKumarNain have a look at https://gitlab.com/raniere-phd/interactive-data-visualisation/-/blob/main/plotly-r.qmd

You can include the HTML as a `iframe`.

Another option is to let plotly generate the HTML code to be included in the HTML by Quarto.

**Reply by AakashKumarNain** [reply]
Thanks @rgaiacs
